src/azurelinux/3.0/net9.0/crossdeps-builder/amd64/Dockerfile

```
@@ -16,7 +16,6 @@ RUN tdnf install -y \
         glibc-devel \
         kernel-headers \
         texinfo \
```

```suggestion
        texinfo \
        zlib-devel \
```
The build errors suggest that the builder crossdeps image needs it for other reasons.